master_pickup:
  state_timer_rate: 10
  
  search_size_x: 10
  search_size_y: 10
  search_spacing: 2
  search_yaw_correction: 0
  search_offset_x: -5
  search_offset_y: -5
  
  takeoff_height: 4
  search_height: 4
  goto_home_tol: 0.25
